src/blog/contributors-youve-earned-a-gitpoap/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Hey Trufflesuite contributors, you’ve earned a POAP!
+hide:
+  - navigation
+---
+
+**By Kingsley Arinze**
+
+Hey Trufflesuite contributors, we’re excited to share that in collaboration with the [GitPOAP](https://gitpoap.io/) team, we will be issuing POAPs in appreciation of your awesome contributions over the years. 
+
+## What are POAPs?
+
+POAPs (Proof Of Attendance Protocol) are badges issued as NFTs (non-fungible token) to prove that the recipient/owner has carried out an action or has attended an event. They are like gold medals from the Olympics, except these exist as digital collectibles. 
+
+## Who's eligible to claim a GitPOAP?
+
+Great question. Annual contributor POAPs will be issued to those who have contributed at least once all year since the start of the following projects:
+
+- Ganache
+- Truffle
+- trufflesuite.com website
+
+Those of you who have contributed to all three projects can claim all three POAPs! 
+
+Here are some POAPs you should expect to receive as a TruffleSuite contributor.
+
+![gitpoap examples](./poap-examples.png)
+
+## How and when can I claim my POAP?
+
+POAPs will be issued as NFTs, so contributors must authenticate with their GitHub accounts on the [GitPOAP website](https://gitpoap.io/) to claim their POAP.
+
+All POAPs are minted on Gnosis Chain (Formerly xDai). There is no cost to claim your POAP and it can be migrated to the Ethereum mainnet if you pay the gas fee for the transaction. Learn how to do that [here](https://intercom.help/poap/en/articles/5790336-how-do-i-migrate-my-poap-to-mainnet).
```

Comment:
"here" is not descriptive and should be avoided in links. This also suggests changing the wording a bit, though I'm not sure if it is much improved after my changes. Feel free to ignore those suggestions. :-)

```suggestion
All POAPs are minted on Gnosis Chain (formerly xDai). There is no cost to claim a POAP and it can be migrated to Ethereum Mainnet by sending a transaction. Visit [how do I migrate my POAP to Mainnet](https://intercom.help/poap/en/articles/5790336-how-do-i-migrate-my-poap-to-mainnet) for instructions.
```